README.md

```diff
@@ -3,6 +3,6 @@
 ### Suggested workflow
  * Check out this repo into e.g. `epoch/eqc`
  * Start a shell using rebar3 (from `epoch/`): `./rebar3 as test shell --apps=""`
- * Move into the directory containing the models you would like to run, e.g.: `1> cd("eqc/aeutils").`
+ * Move into the directory containing the models you would like to run, e.g.: `1> cd("eqc/aeutils_eqc").`
```

Comment:
I see this doc rename included in https://github.com/Quviq/epoch-eqc/commit/e2b1542f77c1511c5fc400861877b0c26bd27193
Doc update proposed as https://github.com/Quviq/epoch-eqc/pull/8